doc/classes/PackedByteArray.xml

```diff
@@ -479,6 +522,29 @@
 				If the original data can't be converted to signed 64-bit integers, the resulting data is undefined.
 			</description>
 		</method>
+		<method name="validate_utf8" qualifiers="const">
+			<return type="int" />
+			<description>
+				Validates UTF-8 encoded array. This function stops after first encountered invalid character (excluding unexpected surrogates). See [method get_string_parse_error] for possible return values.
+			</description>
+		</method>
+		<method name="validate_utf16" qualifiers="const">
+			<return type="int" />
+			<description>
+				Validates UTF-16 encoded array. This function stops after first encountered invalid character (excluding unexpected surrogates). See [method get_string_parse_error] for possible return values.
+			</description>
+		</method>
+		<method name="validate_utf32" qualifiers="const">
+			<return type="int" />
+			<description>
+			</description>
+		</method>
+		<method name="validate_wchar" qualifiers="const">
+			<return type="int" />
+			<description>
+				Validates wide character ([code]wchar_t[/code], UTF-16 on Windows, UTF-32 on other platforms) encoded array. This function stops after first encountered invalid character (excluding unexpected surrogates). See [method get_string_parse_error] for possible return values.
```

Comment:
It sounds okay unwrap this one to make it clearer. (Although I'm not savvy enough on this topic to say if this is appropriate)
```suggestion
				Validates wide character ([code]wchar_t[/code]) encoded array. A wide character is encoded in UTF-16 on Windows and in UTF-32 on all other platforms. This function stops after first encountered invalid character (excluding unexpected surrogates). See [method get_string_parse_error] for possible return values.
```